lib/util/http.ts

```diff
@@ -0,0 +1,26 @@
+import got from './got';
+
+export class Http {
+  hostType: string;
+
+  options: any;
+
+  constructor(hostType: string, options?: any) {
+    this.hostType = hostType;
+    this.options = options;
+  }
+
+  combineOptions(options: any): any {
+    return {
+      json: true,
+      ...this.options,
+      options,
+      hostType: this.hostType,
+    };
+  }
+
+  async get(url: string, options?: any): Promise<any> {
+    const res = await got(url, this.combineOptions(options));
+    return res;
```

Comment:
```suggestion
    return { body: res };
```

Reply:
Maybe we also need headers?

Reply:
`res` already returns `{ body }` in got 9, so I'm confused. Can you explain more?

Reply:
Ups, little typo, fixed it.

We can return full `res` anyway. I want to make sure only our defined  response is used. So no `got` references.